### ISSUE TITLE
Support for EraClean Intelligent deodorizer

### ISSUE THREAD

**Issue by Psytoshgen** [AlexxIT/XiaomiGateway3#1438]
Hi! Your integration got partial support for device
only connected status.
can you add full support of era.airp.cwb03 ?
https://home.miot-spec.com/spec/era.airp.cwb03

i found Esphome component for other model if it helps
https://github.com/dentra/esphome-components/tree/master/components/miot_cwbs01

**Comment by AlexxIT** [comment]
You can try to write converter yourself

**Comment by Psytoshgen** [comment]
> You can try to write converter yourself

I'm not coder, my skills are ctrlC + ctrlV, but i can try, where I can find instructions?

ah yes xiaomi token extractor shows era.airp.cwb03 but instructions of device tells that it era.airp.cwb01 strange things =)

**Comment by dentra** [comment]
It is completely the same model as cwbs01, but with other code.

This device send advertisement of type “online” and it used only to inform about device availability. This device can not work through gateway but via direct BT connection.

**Comment by Psytoshgen** [comment]
> It is completely the same model as cwbs01, but with other code.
> 
> This device send advertisement of type “online” and it used only to inform about device availability. This device can not work through gateway but via direct BT connection.

Yes, works fine with esp32, but there some issues with connection stability, so support with gw3 integration would be great.

**Comment by Psytoshgen** [comment]
> It is completely the same model as cwbs01, but with other code.
> 
> This device send advertisement of type “online” and it used only to inform about device availability. This device can not work through gateway but via direct BT connection.

yes, I use your esp32 component to connect with device, thank you for your work. I think this issue may be closed...
